docs/getting-started-guides/minikube.md

```diff
@@ -267,7 +267,7 @@ We recommend you use `ImagePullSecrets`, but if you would like to configure acce
 
 ## Add-ons
 
-In order to have minikube properly start/restart custom addons, place the addons you wish to be launched with minikube in the `~/.minikube/addons` directory.  Addons in this folder will be moved to the minikubeVM and launched each time minikube is started/restarted.
+In order to have minikube properly start/restart custom addons, place the addons you wish to be launched with minikube in the `~/.minikube/addons` directory.  Addons in this folder will be moved to the minikube VM and launched each time minikube is started/restarted.
```

Comment:
Remove the extra space before "Addons in this ...". 

I would rewrite it as

In order to have minikube properly start or restart custom addons, place the addons you wish to be launched with minikube in the `~/.minikube/addons` directory. Addons in this folder will be moved to the minikube VM and launched each time minikube is started or restarted.